steps/data_preparation_steps/load_data_step/load_data_step.py

```diff
@@ -1,100 +1,39 @@
 """Load data step (data prep pipeline)."""
 
+import os
+from typing import Tuple
+
 import pandas as pd
+from config import DATA_DIR
 from zenml import step
 from zenml.logger import get_logger
-from zenml.post_execution import PipelineView, get_pipeline
-from zenml.post_execution.artifact import ArtifactView
 from zenml.steps import Output
 
 logger = get_logger(__name__)
 
 
-def get_output_from_step(pipeline: PipelineView, step_name: str) -> ArtifactView:
-    """Fetch output from a step with last completed run in a pipeline.
+def _read_data() -> Tuple[pd.DataFrame, pd.DataFrame]:
+    """Read in the data from the data directory.
 
-    Args:
-        pipeline (PipelineView): Post-execution pipeline class object.
-        step_name (str): Name of step to fetch output from.
+    Note: this will need updating in the subsequent integration of DVC proper.
```

Comment:
Just referencing the full implementation of DVC. The description should be covered in the ticket which adds the full implementation, but here's a rough update to the function for @KirsoppJ to add:

```
Note: this will need updating as part of the full implementation of DVC. This includes either pulling the latest version of data or a specific version.
```

Reply:
Ah I see, thanks for clarification.